### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by federicoromeo** [ebrarbilgili/instagram_login#1]
I am experiencing problems with Flutter WebView Plugin

**Comment by yongjieYuan** [comment]
Yes,I tried it and it is still working!I suggest you'd better clone this project.
But there is a small problem in project. Author coded it following document of Instagram,maybe doc has some wrong.
`void getAuthorizationCode(String url) {
    print("AuthorizationUrl: $url");
    authorizationCode = url
        .replaceAll('${InstagramConstant.redirectUri}?code=', '')
        .replaceAll('#/_', ''); /// "#/_"  instead of "#_"
    print("AuthorizationCode: $authorizationCode");
  }`

**Comment by yongjieYuan** [comment]
> I am experiencing problems with Flutter WebView Plugin
I run it with flutter which version is 2.2.3.

**Comment by ebrarbilgili** [comment]
> I am experiencing problems with Flutter WebView Plugin

hello, yes this is still working but it is old version of flutter and packages. maybe new version is different, i dont know...

**Comment by jeeldev9** [comment]
not working flutter_webview_plugin this package not support.
